lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
The value add of this is that the code coverage reports proper line numbers on transpiled code. Without this piece the line numbers are wrong and that makes code coverage metrics a lot less useful.